SkietbaanBE/SkietbaanBE/Models/User.cs

```diff
@@ -1,16 +1,31 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace SkietbaanBE.Models {
-    public class User {
-
+    public class User
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
         public string Username { get; set; }
-        public string Surname { get; set; }
         public string Password { get; set; }
```

Comment:
This password field should ideally be stored as a byte[] since passwords must be hashed when stored in the db

Reply:
noted with thanks